ee/clickhouse/queries/trends/clickhouse_trends.py

```diff
@@ -52,6 +53,45 @@ def _run_query(self, filter: Filter, entity: Entity, team_id: int) -> List[Dict[
             serialized_data = self._handle_cumulative(serialized_data)
         return serialized_data
 
+    def _run_query_for_threading(self, result: List, index: int, sql, params):
+        result[index] = sync_execute(sql, params)
+
+    def _run_parallel(self, filter: Filter, team: Team) -> List[Dict[str, Any]]:
+        result: List[Union[None, List[Dict[str, Any]]]] = [None] * len(filter.entities)
+        jobs = []
+
+        for entity in filter.entities:
+            sql, params, _ = self._get_sql_for_entity(filter, entity, team.pk)
+            thread = threading.Thread(target=self._run_query_for_threading, args=(result, entity.index, sql, params),)
+            jobs.append(thread)
+
+        # Start the threads (i.e. calculate the random number lists)
```

Comment:
Q: Why do we have a separate loop for starts? Why not start it after line 65?

Reply:
Good question! I tried this but turns out doing this will run everything sequentially